main.cpp

```diff
@@ -108,8 +108,54 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Person
+{
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTraveled;
+
+    void run(int howFast, bool startWithLeftFoot);
+};
 
+struct Foot
+{
+    int size = 100;
+    std::string currentPosition;
+    
+    void stepForward();
+    int setSize();
+};
+
+void Foot::stepForward()
+{
+    currentPosition = "Forward";
+}
 
+int Foot::setSize()
+{
+    return size;
+}
+
+void Person::run(int howFast, bool startWithLeftFoot)
+{
+    Foot leftFoot;
+    Foot rightFoot;
+    howFast = 25;
```

Comment:
these `Foot` instances are member variables.  They were not declared inside this function in the video.
Restructure your code accordingly.
- [ ] revise